doc/running/index.rst

```diff
@@ -186,7 +186,7 @@ This will generate, compile, build, and install the code for a set of specified
    * - ``--target_path``
      - (Optional) Path to target directory where generated code will be written into. Default is ``target``, which will be created in the current working directory if it does not yet exist.
    * - ``--target_platform``
-     - (Optional) The name of the target platform to generate code for. Default is ``NEST``.
+     - (Optional) The name of the target platform to generate code for. The available targets are ``NEST`` and ``NEST_DESKTOP``. Default is ``NEST``.
```

Comment:
Python and compartmental and SpiNNaker seem to be missing from the list. Could we just link back to the paragraph "Supported target platforms" (https://nestml.readthedocs.io/en/latest/running/index.html#supported-target-platforms)?